_includes/gcp-gke-prereqs.md

```diff
@@ -4,8 +4,12 @@
 1. [Install Node.js version 6 or later](https://nodejs.org/en/download/)
 1. Install a package manager for Node.js, such as [npm](https://www.npmjs.com/get-npm) or [Yarn](https://yarnpkg.com/en/docs/install).
 1. [Install Google Cloud SDK (`gcloud`)](https://cloud.google.com/sdk/docs/downloads-interactive)
-1. Configure Auth Options
-    1. `gcloud` Login
+1. Configure GCP Auth
+
+    Choose **either option** below to configure your local machine with GCP
```

Comment:
I think this is fine. I do wonder, however, if we should even be presenting an option. An alternative is to do something more like:

Login using `gcloud`

```bash
... etc etc etc ...
```

> Note that this auth mechanism is meant for inner loop developer workflows. If you want to run this example in an unattended service account setting, such as in CI/CD, please [follow instructions to configure your service account].

I can just still see people going down the rabbit's hole of (2) and for a tutorial, I suspect it is seldom needed.